pkg/azurefile/nodeserver.go

```diff
@@ -59,6 +59,19 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 	mountPermissions := d.mountPermissions
 	context := req.GetVolumeContext()
 	if context != nil {
+		// token request
+		if context[serviceAccountTokenField] != "" && context[clientIDField] != "" {
```

Comment:
`clientID` is a user input parameter, since all user input parameters are not case sensitive, suggest using a func to check whether it's a token request

Reply:
I use a func to check the clientIDField for both upper and lower case.